.ado/pipelines/config/configuration.yaml

```diff
@@ -5,15 +5,15 @@ variables:
   value: 'foundational-connected'
 
 - name: 'terraformVersion'    # Terraform Version
-  value: '1.1.6'
+  value: '1.1.7'
 - name: 'kubernetesVersion'   # kubernetes version used for aks clusters
   value: '1.22.6'
 - name: 'helmVersion'         # helm package manager version
-  value: 'v3.8.0'
+  value: 'v3.8.1'
 - name: 'ingressNginxVersion' # nginx ingress controller helm chart version
   value: '4.0.16'
 - name: 'certManagerVersion'  # cert-manager helm chart version
-  value: 'v1.6.2'
+  value: 'v1.7.1'
```

Comment:
@sebader have you already tested if 1.7.1 works? Had some trouble when upgrading to 1.6.x haven't played with 1.7.x yet.

Reply:
Yes, worked without issues. What did you see?

Reply:
Manifests requiring updates / new api versions used etc. but might be only the case in the last minor release. Haven't read the release notes for any of the 1.7.x releases, yet.

Reply:
Interesting. It's running in the private-aks branch (with DNS issuer) in this env